charts/timed/templates/service-postgresql.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.postgresql.ingress.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+  labels:
+    {{- include "timed.labels" . | nindent 4 }}
+    app: postgresql
+spec:
+  type: LoadBalancer
+  ports:
+    - port: 5432
+      targetPort: 5432
+      protocol: TCP
+      name: tcp-postgresql
+  selector:
+    app: postgresql
+    role: slave
```

Comment:
Same here although it looks like the upstream postgresql chart is already using legacy labels.

Maybe the `timed.selectorLabels` named template helper can also be part of this?

To account for deployments that do not use the postgresql chart as a dependency the deployment might need to have a possibility to override the labels.